packages/boxed-expression-component/src/table/BeeTable/BeeTableContextMenuHandler.tsx

```diff
@@ -367,21 +393,21 @@ export function BeeTableContextMenuHandler({
     ]
   );
 
-  const onMinus = () => {
+  const onMinus = useCallback(() => {
     const newValue = (insertMultipleRowColumnsValue || 0) - 1;
     setInsertMultipleRowColumnsValue(newValue);
-  };
+  }, [insertMultipleRowColumnsValue]);
```

Comment:
```suggestion
  const onMinus = useCallback(() => {
    setInsertMultipleRowColumnsValue(prev => (prev || 0) - 1);
  }, []);
```

Avoid a dependency on the useCallback :) Also, why not make the Math.min and Math.max here at those, instead of at `onChange`?

Reply:
Same applies for `onChange` and `onPlus`. btw